<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Vani&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="piecesBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>71</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Vani&quot;;
color: rgb(144, 0, 0);</string>
    </property>
    <property name="title">
     <string>Pieces</string>
    </property>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>15</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="resourceCardsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Vani&quot;;
color: rgb(144, 0, 0);</string>
    </property>
    <property name="title">
     <string>Resource Cards</string>
    </property>
    <widget class="QLabel" name="numOfBricksLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="numOfLumbersLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="numOfOresLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="numOfWoolsLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="numOfWheatsLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Algerian&quot;;
color: rgb(209, 0, 0);
background-color: rgb(255, 205, 55);
border-color: rgb(255, 205, 55);
</string>
    </property>
    <property name="text">
     <string>Trade</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Favorites/Desktop/catan images/trade/trade.png</normaloff>../../Favorites/Desktop/catan images/trade/trade.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Algerian&quot;;
color: rgb(209, 0, 0);
background-color: rgb(255, 205, 55);
border-color: rgb(255, 205, 55);
selection-background-color:rgb(255, 205, 55);</string>
    </property>
    <property name="text">
     <string>End your turn</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Favorites/Desktop/catan images/trade/next.jpg</normaloff>../../Favorites/Desktop/catan images/trade/next.jpg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="playerPhotoLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Favorites/Desktop/catan images/player photo/11.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerUsernameLine">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 214, 110);
font: 9pt &quot;Vani&quot;;
color: rgb(144, 0, 0);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Vani&quot;;
color: rgb(144, 0, 0);</string>
    </property>
    <property name="title">
     <string>Development Cards</string>
    </property>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="numOfLumbersLabel_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="numOfWoolsLabel_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>51</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="numOfOresLabel_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="numOfBricksLabel_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="numOfWheatsLabel_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/catan-logo.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
